### ISSUE TITLE
Spatialite trying to copy geom from existing object

### ISSUE THREAD

**Issue by corystone** [geoalchemy/geoalchemy2#239]
Hi folks, I'm trying to do something that seems simple (but maybe weird) but doesn't seem to work.

Anyway, this is straight out of the tutorial:

```python
from geoalchemy2 import Geometry
from sqlalchemy import create_engine, Column, Integer, String
from sqlalchemy.event import listen
from sqlalchemy.sql import select, func
from sqlalchemy.ext.declarative import declarative_base
from sqlalchemy.orm import sessionmaker


def load_spatialite(dbapi_conn, connection_record):
    dbapi_conn.enable_load_extension(True)
    dbapi_conn.load_extension("/usr/lib/x86_64-linux-gnu/mod_spatialite.so")


engine = create_engine("sqlite:///example.db", echo=True)
listen(engine, "connect", load_spatialite)

conn = engine.connect()
conn.execute(select([func.InitSpatialMetaData(1)]))
conn.close()

Base = declarative_base()


class Lake(Base):
    __tablename__ = "lake"
    id = Column(Integer, primary_key=True)
    name = Column(String)
    geom = Column(Geometry(geometry_type="POLYGON", management=True))


Lake.__table__.create(engine)

Session = sessionmaker(bind=engine)
session = Session()

lake = Lake(name="Majeur", geom="POLYGON((0 0,1 0,1 1,0 1,0 0))")
session.add(lake)
session.commit()

new_lake = Lake(name='somethingelse', geom=lake.geom)
session.add(new_lake)
session.commit()
```

```
2019-06-21 20:02:41,786 INFO sqlalchemy.engine.base.Engine INSERT INTO lake (name, geom) VALUES (?, GeomFromEWKB(?))
2019-06-21 20:02:41,787 INFO sqlalchemy.engine.base.Engine ('somethingelse', '0103000020FFFFFFFF010000000500000000000000000000000000000000000000000000000000F03F0000000000000000000000000000F03F000000000000F03F0000000000000000000000000000F03F00000000000000000000000000000000')
2019-06-21 20:02:41,788 INFO sqlalchemy.engine.base.Engine ROLLBACK
```

snip

```
sqlite3.IntegrityError: lake.geom violates Geometry constraint [geom-type or SRID not allowed]
```

Doing a little digging, the geometry object from GeomFromEWKB has an SRID of 0, while spatialite wants it to be a -1 to be valid.

Is this expected?

I am currently running:
Python 3.7.3
GeoAlchemy2==0.6.3
SQLAlchemy==1.3.5
Shapely==1.6.4.post2

**Comment by elemoine** [comment]
Confirmed. I'll have a closer look.

In the mean time please try passing an srid to the geometry type and wkt elements:

```
geom = Column(Geometry(geometry_type="POLYGON", management=True, srid=4326))
```

```
lake = Lake(name="Majeur", geom=WKTElement("POLYGON((0 0,1 0,1 1,0 1,0 0))", srid=4326))
```

**Comment by adrien-berchet** [comment]
This issue seems to be fixed in `geoalchemy2 >= 0.8`.
Please reopen if you still encounter it.
